src/library/Uno.Material/Styles/Controls/ListView.xaml

```diff
@@ -0,0 +1,73 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+
+	<ResourceDictionary.MergedDictionaries>
+		<MaterialFonts xmlns="using:Uno.Material" />
+		<MaterialColors xmlns="using:Uno.Material"/>
+	</ResourceDictionary.MergedDictionaries>
+
+	<Style x:Key="MaterialListViewItemStyle" TargetType="ListViewItem">
+		<Setter Property="FontFamily" Value="{StaticResource MaterialFontFamily}" />
+		<Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}" />
+		<Setter Property="Background" Value="{ThemeResource ListViewItemBackground}" />
+		<Setter Property="Foreground" Value="{ThemeResource ListViewItemForeground}" />
+		<Setter Property="TabNavigation" Value="Local" />
+		<Setter Property="IsHoldingEnabled" Value="True" />
+		<Setter Property="Padding" Value="12,0,12,0" />
+		<Setter Property="HorizontalContentAlignment" Value="Stretch" />
+		<Setter Property="VerticalContentAlignment" Value="Center" />
+		<Setter Property="MinWidth" Value="{ThemeResource ListViewItemMinWidth}" />
+		<Setter Property="MinHeight" Value="{ThemeResource ListViewItemMinHeight}" />
+		<Setter Property="AllowDrop" Value="False" />
+		<Setter Property="UseSystemFocusVisuals" Value="True" />
+		<Setter Property="FocusVisualMargin" Value="0" />
+		<Setter Property="FocusVisualPrimaryBrush" Value="{ThemeResource ListViewItemFocusVisualPrimaryBrush}" />
+		<Setter Property="FocusVisualPrimaryThickness" Value="2" />
+		<Setter Property="FocusVisualSecondaryBrush" Value="{ThemeResource ListViewItemFocusVisualSecondaryBrush}" />
+		<Setter Property="FocusVisualSecondaryThickness" Value="1" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="ListViewItem">
+					<ListViewItemPresenter ContentTransitions="{TemplateBinding ContentTransitions}"
```

Comment:
ListViewItemPresenter is not supported on Uno, so this would not actually show anything - https://github.com/unoplatform/uno/issues/1444

you can use `BasedOn="ListViewItemExpanded"` or copy the template from ListViewItemExpanded - https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml#L959

(note that it exists on UWP also, so BasedOn is safe)